File: 1013.c

#include <stdio.h>
#include <stdlib.h>

int main(void){
    int a, b, c, resultado;
    scanf("%d %d %d", &a, &b, &c);
    resultado = (a+b+abs(a-b))/2;
    printf("%d eh o maior\n", (resultado+c+abs(resultado-c))/2);
    return 0;
}

File: 1013.py
#!python3.8

# adriano r. de sousa

values = [int(x) for x in input().split()]

highest_value = (values[0]+values[1]+abs(values[0]-values[1]))/2

highest_value = (highest_value+values[2]+abs(highest_value-values[2]))/2

print(f'{int(highest_value)} eh o maior')
